tests/onnx/test_onnx_v2.py

```diff
@@ -262,6 +263,8 @@ def _onnx_export(self, test_name, name, model_name, feature, onnx_config_class_c
 
         model_class = FeaturesManager.get_model_class_for_feature(feature)
         config = AutoConfig.from_pretrained(model_name)
+        if model_name == "microsoft/deberta-v2-xlarge":
+            config.type_vocab_size = 2
```

Comment:
```suggestion
        if model_name == "microsoft/deberta-v2-xlarge":
            config = AutoConfig.from_pretrained(model_name, type_vocab_size=2)
        else:
            config = AutoConfig.from_pretrained(model_name)
```

Reply:
@JingyaHuang Is this how you would prefer I have type vocab size on for the test?

Reply:
@sam-h-bean You can do it like that for your own test, to confirm that the export works and in the exported IR there will be the embedding of token type. However, in the transformers test it looks a little bit hacky to set a conditional statement for deberta models only, as deberta is not the only one can have `token_type_ids`. I would prefer that you remove these line from the test in transformers.

Reply:
Removed!